src/site/content/speakers/hen-peretz.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Zen and The Art of Beer Fermentation Observability
+description: "While all you need in order to brew your own beer is just a big pot and a stirring spoon, IoT devices are everywhere and the beer industry doesn’t stand behind. We are going to play and build a serverless application to keep track of beer fermentation and learn how to monitor serverless applications"
+speaker: Hen Peretz
+bio: "While all you need in order to brew your own beer is just a big pot and a stirring spoon, IoT devices are everywhere and the beer industry doesn’t stand behind. We are going to play and build a serverless application to keep track of beer fermentation and learn how to monitor serverless applications"
+image: https://res.cloudinary.com/dsgynmxwi/image/upload/v1550771145/serverlessdays%20amsterdam/hen-peretz.jpg
```

Comment:
We can store images in the site itself if we want. They will be deployed automatically (see Yan's page).